### DISCUSSION TITLE
Should CI tests run on push to main?

### DISCUSSION THREAD

**Discussion by gwhitney** [numberscope/frontscope#213] [Ideas]
Most github repositories show a check when the current contents of main pass the ci tests. Currently ours does not. Perhaps it is because the CI actions do not run on on push, only on pull requests. I like not having redundant runs on each push to a branch and then on the PR from that branch, but I would not mind having the (admittedly slightly redundant) check on main. Is there a way to configure Github actions so that the tests will run after each merge to main and then be shown on the github home page for the repo? If so, would we want to do that?

**Comment by liammulh** [comment]
I think the CI actions can be configured to run on push to main. Something like:

```
on:
  push:
    branches:
      - main
```

**Comment by liammulh** [comment]
I'm fine with doing that as long as we don't run out of free minutes.

**Reply by gwhitney** [reply]
I think right now we're nowhere near using up the free tier. I will convert this into an issue.

**Comment by liammulh** [comment]
More detailed example on this page: https://docs.github.com/en/actions/examples/using-scripts-to-test-your-code-on-a-runner.

**Comment by gwhitney** [comment]
As this is now implemented, closing.
